### PR TITLE
Configure changesets

### PR DESCRIPTION
**Pull request by ludovicm67**
This PR does the following:
- start structuring the repo to be a mono-repo ; this will help us a lot for maintaining the whole project,
- configure linter at the root
- configure changesets for releasing packages

### CODE REVIEW

File: packages/trifid/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.idea
+coverage
+node_modules
+npm-debug.log
+package-lock.json
```
Comment (tpluscode) [inline packages/trifid/.gitignore:5]: don't suppose you need nested ignore file

File: packages/trifid/.npmignore
Comment (tpluscode) [inline packages/trifid/.npmignore:1]: is it needed in package dir?

File: packages/trifid/.npmrc
Comment (tpluscode) [inline packages/trifid/.npmrc:1]: ditto

File: packages/trifid/.npmignore
Comment (ludovicm67) [reply packages/trifid/.npmignore:1]: For now, yes. This avoids packing useless files. The ideal way would be to whitelist files using the `files` property of the `package.json` file. I will take care of this in a future commit.

**Pull request merged**